lib/rules/attributes-order.js

```diff
@@ -20,6 +20,8 @@ const ATTRS = {
   TWO_WAY_BINDING: 'TWO_WAY_BINDING',
   OTHER_DIRECTIVES: 'OTHER_DIRECTIVES',
   OTHER_ATTR: 'OTHER_ATTR',
+  OTHER_ATTR_STATIC: 'OTHER_ATTR_STATIC',
+  OTHER_ATTR_DYNAMIC: 'OTHER_ATTR_DYNAMIC',
```

Comment:
I'm wondering if the new options need the `OTHER_` prefix, or if we can rename them to `ATTR_DYNAMIC` and `ATTR_STATIC`. What's your opinion?

Reply:
I removed all `OTHER_` prefix, it really got better